src/frontend/js/components/Form/test-utils.ts

```diff
@@ -0,0 +1,25 @@
+import { fireEvent, within } from '@testing-library/dom';
+import { act } from '@testing-library/react';
+import userEvent, { UserEvent } from '@testing-library/user-event';
+
+export const changeSelect = async (
+  $input: HTMLElement,
+  value: string,
+  user: UserEvent = userEvent.setup(),
+) => {
+  await userEvent.click($input);
+  const $selectListBox = within($input.closest('.c__field')!).getByRole('listbox');
+  await user.click(
+    within($selectListBox).getByRole('option', {
+      name: value,
+    }),
+  );
+};
+
+export const clearSelect = async ($input: HTMLElement) => {
+  await act(async () => {
+    fireEvent.click(
```

Comment:
My bad.
My first solution was to use await act fireEvent because i didn't succeed to have it work with userEvent.
Then i refactor everything removing act/fireEvent and forgot this one.